app/routes/caseload.js

```diff
@@ -10,10 +10,6 @@ module.exports = function (router) {
     var organisationLevel = req.params.organisationLevel
     var id = req.params.id
 
-    if (organisationLevel === organisationUnitConstants.OFFENDER_MANAGER.name) {
```

Comment:
Probably still need this error in case someone manually enters the URL

Reply:
Added back in